mne/io/meas_info.py

```diff
@@ -1342,16 +1351,27 @@ def _merge_info(infos, force_update_to_first=False, verbose=None):
             msg = ("Measurement infos provide mutually inconsistent %s" %
                    trans_name)
             raise ValueError(msg)
+
+    # KIT system-IDs
+    kit_sys_ids = [i['kit_system_id'] for i in infos if i['kit_system_id']]
+    if len(kit_sys_ids) == 0:
+        info['kit_system_id'] = None
+    elif len(kit_sys_ids) == 1:
+        info['kit_system_id'] = kit_sys_ids[0]
+    else:
+        raise ValueError("Trying to merge channels form different KIT systems")
```

Comment:
This would happen if infos had the same system id. Maybe something like 
```
elif len(set(kit_sys_ids)) == 1:
    ...
else:
    raise ...
```
Also, form -> from